pages/_app.js

```diff
@@ -1,7 +1,14 @@
 import '../styles/globals.css'
+import Header from '../templates/Header'
+
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+  return (
+    <>
+      <Header />
```

Comment:
Create a layouts folder and create a AppLayout component and place your header there

```
const AppLayout = ({children}) => {

return (
  <Grid>
   <Header />
   <Grid>
    {children}
   </Grid>
 </Grid>
 )
}
 ```

Reply:
After which, you wrap all your app screens with this layout